js/navbar.js

```diff
@@ -0,0 +1,55 @@
+export function navbar() {
+  const allButtons = document.querySelectorAll('[data-nav]');
+  const allPages = document.querySelectorAll('[data-page]');
+
+  allButtons.forEach(button => {
+    button.addEventListener('click', event => {
+      allPages.forEach(page => {
+        page.classList.add('hidden');
+      });
+      const buttonAttribute = event.currentTarget.getAttribute('data-nav');
+      const currentPage = document.querySelector(
+        `[data-page="${buttonAttribute}"]`
+      );
+      currentPage.classList.remove('hidden');
+    });
+  });
+
+  const buttonHome = document.querySelector('[data-nav="home"]');
```

Comment:
Good job with including the short version - so you can take those lines below  out right? 😊
or did i oversee something?😊

Reply:
Thank you! I am still surprised, that it worked without a problem. But I didn´t figure out yet, how to change the buttons to active in the same manner. So I used my old version for the buttons just to make it work. In my next branch I am going to work on that problem. Thank you very much for your nice review! 😸